src/components/events/DetailsAddons.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <DetailsBase class="details-addons">
+    <template #header>
+      ADDONS
+    </template>
+    <template #content>
+      <div
+        v-for="(value, key) in addons"
+        :key="key"
+        class="event-details__content-block"
+      >
+        <div class="event-details__key">
+          {{ key }}
+        </div>
+        <div class="event-details__value">
+          {{ renderAddonValue(key, value) }}
+        </div>
+      </div>
+    </template>
+  </DetailsBase>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import DetailsBase from './DetailsBase.vue';
+
+/**
+ * Details addons component
+ */
+export default Vue.extend({
+  name: 'DetailsAddons',
+  components: {
+    DetailsBase,
+  },
+  props: {
+    addons: {
+      type: Object,
+      required: true,
+    },
+  },
+  methods: {
+    /**
+     * Render value in correct format
+     * @param {string | object} key - addons key
+     * @param {string} value - addons value
+     * @return {String}
+     */
+    renderAddonValue(key, value): string {
```

Comment:
нужно указать типы параметров в коде
```suggestion
    renderAddonValue(key: string | object, value: string): string {
```